BaragonCore/src/main/java/com/hubspot/baragon/models/BaragonSource.java

```diff
@@ -0,0 +1,51 @@
+package com.hubspot.baragon.models;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.Optional;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class BaragonSource {
+  private final String name;
+  private final Optional<Integer> port;
+
+  @JsonCreator
```

Comment:
Gotcha, thanks. I appreciate the rigor, but it'd be simplest to hold off on a ZK migration until it's explicitly needed.